settings/provisioner/distro/rhel-7.2.yml

```diff
@@ -0,0 +1,8 @@
+image:
+  file: rhel-guest-image-7.2-20160302.0.x86_64.qcow2
+  server: "{{ !lookup private.images.rhel.7.2.remote_file_server }}"
+name: rhel
```

Comment:
It's just a common information about the distro which is currently not used. But the distro yaml can be used by different apps. 

I can remove it but I think we would need that data later.